### ISSUE TITLE
Add support for Diff as a format

### ISSUE THREAD

**Issue by mmahmoudian** [PrivateBin/PrivateBin#869]
## Brief background

I typically use PrivateBin to share logs, config files or snippets of codes, and other than the log file, having a diff format in which I can paste two texts and it shows the diff would be pretty useful. At the time being I'm using a proprietary website (diffchecker.com) to do this, but I feel this is to the interest of FOSS in general to have such feature.

## General gist of the idea

It would be a nice addition to have Diff as a format:

![image](https://user-images.githubusercontent.com/390889/149923927-bbf20172-66d2-40d3-b48e-2845e1f3ce4f.png)

Example from a proprietary platform:
https://www.diffchecker.com/ex5lCdlC

![image](https://user-images.githubusercontent.com/390889/149924580-2360ff2e-11e0-456e-b3f6-81cbc36d45bc.png)

**Comment by rugk** [comment]
> This would be in-line with #189 but there are obvious differences. I believe these two are complementary as #189 talk about a single diff patch file, and I here propose of having two input boxes to insert the two texts and PrivateBin calculate the Diff itself and then show it in a visual way.

Okay, then it really is no duplicate, however a really advanced and very specific feature/use case. After all, if you want to send a diff of a paste, you can use any diff tool of your website and then copy/follow that into the source code view if https://github.com/PrivateBin/PrivateBin/issues/189 is implemented.

**Comment by mmahmoudian** [comment]
Yes not a duplicate, but very close concepts. 😅 I didn't want to highjack #189 thread, so I just opened this issue.

Regarding the diff though, for tech-savvy people you are right that the output of diff command and #189 would do it justice probably (the only problem is the number of context lines are 3 by default), but for the general population who want to show something has changed in a text file (before/after) when they write an assignment or homework, this side-by-side feature would be user friendly imho.
